.github/actions/ngen-build/action.yaml

```diff
@@ -183,8 +183,11 @@ runs:
         - name: Build Targets
           #cmake >= 3.15 can build multiple targets
           run: |
+            # Build Targets
```

Comment:
Added this so that the folded `run:` line in the Github Action would show something more meaningful than the comment on the next line